web/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:18
+
+# Create app directory
+WORKDIR /usr/src/app
+
+# Install app dependencies
+# A wildcard is used to ensure both package.json AND package-lock.json are copied
+# where available (npm@5+)
+COPY package*.json ./
+
+RUN npm install
+# If you are building your code for production
+# RUN npm ci --only=production
+
+# Bundle app source
+COPY . .
+
+EXPOSE 8080
+CMD [ "npm", "start" ]
```

Comment:
I believe we need to run `npm run build` in here somewhere

Reply:
Ah, okay, that's why it wasn't working for me. It was using a locally built version from a previous build. When I cleaned everything out and rebuilt, it worked.